docker/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN apt-get update \
       automake \
```

Comment:
Should we also roll the stateless/base build to :10 while we are in here?  (Several lines above this.)
Might want to do that as a separate PR, though.

Reply:
In a effort to not cause a holy war, could we just use the gobgp from osrg?